static/js/settings_bots.js

```diff
@@ -99,6 +104,10 @@ exports.set_up = function () {
     $('#config_inputbox').children().hide();
     $("[name*='"+selected_embedded_bot+"']").show();
 
+    $('#bot_type_'+BOT_TYPES.GENERIC+'_radio').prop('checked', true);
+    $('#bot_type_'+BOT_TYPES.INCOMING_WEBHOOK+'_radio').closest('label')
+        .prop('title', "Incoming webhooks can only send messages.");
```

Comment:
(1) This relocated string isn't tagged for translation

(2) I think it'd be better to have the title attribute live in the template, not be dynamically edited here; that's a much more natural way that one would expect to read the code.

Reply:
(I'd rather have the loop in the template unrolled and have some semi-duplicate code there than add more code hacking up the HTML from JS)